src/ipa/rpi/pisp/data/imx219.json

```diff
@@ -1182,6 +1182,15 @@
                     ]
                 }
             }
-        }
+        },
+          {  "rpi.sync":
```

Comment:
Generally the indentation in all these new json additions doesn't seem quite to match what we have every else. Can we fix that?